src/rai/rai/cli/rai_cli.py

```diff
@@ -38,24 +38,20 @@ def parse_whoami_package():
         description="Parse robot whoami package. Script builds a vector store, creates a robot identity and a URDF description."
     )
     parser.add_argument(
-        "documentation_root", type=str, help="Path to the root of the documentation"
-    )
-    parser.add_argument(
-        "--output",
-        type=str,
-        required=False,
-        default=None,
-        help="Path to the output directory",
+        "whoami_package_root", type=str, help="Path to the root of the whoami package"
     )
+
     args = parser.parse_args()
-    save_dir = args.output if args.output is not None else args.documentation_root
+    save_dir = Path(args.whoami_package_root) / "description" / "generated"
```

Comment:
Ideally, this should adhere to the source / build divide and land in the installation /share folder. I understand there's an issue of cost and, to a lesser degree, build time.  But this should not be in sources most likely

Reply:
I actually think otherwise. User should be able to modify the generated files to fine-tune them. Creating new identity/vector database every run could increase non-deterministic behaviors.

Reply:
Keeping it as a build artifact does not mean creating a new identity vector database every run, correct?

Reply:
Correct. Either way I would like to give the users ability to view, modify and git these files. I can agree to a certain point that these files could be seen as build artifacts but I believe, that the current solution is better. Adhering to ROS 2 standard in this case reduces the user friendliness.